### PR TITLE
Remove polyfills: we don't support IE anymore

### PR DESCRIPTION
**Pull request by askvortsov1**

### CODE REVIEW

File: js/src/forum/index.js [hunk 1]
```diff
@@ -30,9 +28,13 @@ app.initializers.add('flarum-markdown', function(app) {
...
-    const editor = new MarkdownArea(element);
-    editor.disableInline();
-    editor.ignoreTab();
+    const editor = new MarkdownArea(element, {
+      keyMap: {
+        indent: ['Ctrl+m'],
+        outdent: ['Ctrl+M'],
+        inline: ['"', "'", '`']
+      }
+    });
```
Comment (dsevillamartin) [inline js/src/forum/index.js]: This should not be part of this PR. It is a bugfix that changes functionality.

Comment (dsevillamartin) [reply js/src/forum/index.js]: Oh, I see you already have a PR for it @ https://github.com/flarum/markdown/pull/20.

Comment (askvortsov1) [reply js/src/forum/index.js]: Not sure how that got in there... Reverted

**Review by dsevillamartin — APPROVED** [review]

**Review by tankerkiller125 — APPROVED** [review]
